source/functions.txt

```diff
@@ -25,649 +25,11 @@ Atlas Functions
    :hidden:
 
    Query MongoDB Atlas </functions/mongodb>
-   Context </functions/context>
-   Global Modules </functions/globals>
-   External Dependencies </functions/dependencies>
    Test Functions </functions/test>
-   Handle Errors </functions/handle-errors>
-   JavaScript Support </functions/javascript-support>
 
-.. banner::
-   :variant:  warning
+As part of the deprecation of Atlas App Services, Functions will only be
+available through Triggers. The Functions documentation has been moved to
```

Comment:
It's a bit more nuanced. Something that explains a bit more of the why would be good. For example:
> As of September 2024, the Atlas Device SDKs and HTTPS Endpoints are deprecated and will reach end of life in September 2025. Due to this deprecation, Functions will mostly be used with Triggers. For that reason, the Functions documentation has been moved...

Functions and other ancillary services will still be available through the `View All Apps` button and the App view. But we don't really want to encourage creating new standalone Functions because they won't have any way to call them when the primary services go offline.

Reply:
Applies to the values and secrets page too. We should probably add a banner or something to the Authentication section explaining the same thing. I can do that in another PR if you'd like.

Reply:
Got it - I've updated these pages. Does this look more in line with what we want to communicate, @krollins-mdb ?

I haven't worked with banners before and am not sure what exactly the syntax looks like or where you'd like to apply one. Feel free to add a commit here if you want to merge it with this PR, or we could do it as a separate PR.